galaxy_ng/app/migrations/0030_update_collection_remote_rhcertified_url.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+
+def update_collection_remote_rhcertified_url(apps, schema_editor):
+    """
+    Updates the existing collection remote `rh-certified` url field  
+    if startswith `https://cloud.redhat.com/`.
+    """
+    
+    CollectionRemote = apps.get_model('ansible', 'CollectionRemote')
+    
+    rh_remote = CollectionRemote.objects.get(name='rh-certified')
+
+    if rh_remote and rh_remote.url.startswith('https://cloud.redhat.com/'):
+        rh_remote.url = rh_remote.url.replace('https://cloud.redhat.com/', 'https://console.redhat.com/')
+        rh_remote.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('galaxy', '0028_update_synclist_model'),
```

Comment:
now that the latest migration starts with 0029, this dependency can be updated with that